data/test/scenarios/event_handlers_in_events.cfg

```diff
@@ -48,6 +79,17 @@
     [/event]
 )}
 
+#####
+# API(s) being tested: [event][fire_event],[event][event]
+##
+# Actions:
+# Define a custom event nested within another event
+# The nested event is defined second in the WML
+# Fire the custom event from within the same event that the custom event was defined in with [fire_event] using its name
```

Comment:
It's not "within the same event that the custom event was defined in".

This three lines might be clearer as
* Attempt to fire a custom event before that custom event has been defined
* Define the custom event